qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -5,35 +5,24 @@
 #
 
 from test_framework.test_framework import ComparisonTestFramework
-from test_framework.util import *
+from test_framework.util import start_nodes
```

Comment:
#2450 will make that even more annoying, but it's worth it so that the Python code is actually maintainable.